crates/shared/src/bad_token/token_owner_finder/seasolver.rs

```diff
@@ -0,0 +1,45 @@
+use super::TokenOwnerProposing;
+use anyhow::Result;
+use ethcontract::H160;
+
+pub struct SeasolverTokenOwnerFinder {
```

Comment:
I think deriving `Debug`, `Clone` and `Default` is probably not a bad idea.

Reply:
Indeed, however I thought our general stand is that we should not add derives we don't use.